wordpress-mobile/WordPress-Android
# Add log encryption

Implements log encryption.

Replaces the former implementation (which used GPLv3), which we're not able to use due to license incompatibility. 

**To test:**
- Ensure CI passes
- Manually run the `androidTest/util/EncyptionUtilsTest` test suite, which covers this code.

**To review:**
I've added comments in a few places for discussion or validation.

**PR submission checklist:**
